deps/build.jl

```diff
@@ -1,24 +1,35 @@
 using Compat
+
 @static if VERSION >= v"0.7"
     using Libdl
 end
 
-depsfile = joinpath(dirname(@__FILE__),"deps.jl")
-if isfile(depsfile)
-    rm(depsfile)
+const DEPS_FILE = joinpath(@__DIR__, "deps.jl")
+
+if isfile(DEPS_FILE)
+    rm(DEPS_FILE)
 end
 
 function write_depsfile(path)
-    f = open(depsfile,"w")
-    println(f,"const libgurobi = \"$path\"")
-    close(f)
+    # When `path` gets written out to a file, it will escape any backslashes, so
+    # we need to doubly escape them. If your path uses forward slashes, this
+    # operation won't do anything.
+    path = replace(path, "\\" => "\\\\")
```

Comment:
What happens if there are backslashes in a unix path (i.e. the backslashes are part of the name of a folder and do not represent a separation), should we make the replace or not ? That is, should we do this on Windows only ?